### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by JustNrik**
## Summary

Interfaces don't implement each other. They **inherit** each other.

### CODE REVIEW

File: docs/csharp/programming-guide/interfaces/index.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ The following example shows an implementation of the <xref:System.IEquatable%601
...
+Interfaces can inherit other interfaces. A class might include an interface multiple times through base classes that it inherits or through interfaces that other interfaces implement. However, the class can provide an implementation of an interface only one time and only if the class declares the interface as part of the definition of the class (`class ClassName : InterfaceName`). If the interface is inherited because you inherited a base class that implements the interface, the base class provides the implementation of the members of the interface. However, the derived class can reimplement the interface members instead of using the inherited implementation.
```
Comment (mairaw) [inline docs/csharp/programming-guide/interfaces/index.md]: I think you're right based on what the spec says (https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/interfaces#base-interfaces). I think it'd just be better to say can inherit **from** other interfaces.

Comment (JustNrik) [reply docs/csharp/programming-guide/interfaces/index.md]: done :smile:

**Comment by BillWagner** [comment]
@JustNrik Thanks for opening this, and updating it based on @mairaw 's comments. What you've done has made this better, and has made me notice a couple of issues unrelated to your PR in the same paragraph.  Let me know if you want to address these as well in the same PR. If not, I'll open an issue and fix them myself:

> A class might include an interface multiple times through base classes that it inherits or through interfaces that other interfaces implement. 

The end of this sentence should also be "other interfaces implement."

> However, the derived class can reimplement the interface members instead of using the inherited implementation.

This is only true if the base class marked those members as `virtual`. I'd re-word it as:

> However, the derived class can reimplement any virtual interface members instead of using the inherited implementation.

Yes, that change uses the some weasel words and doesn't cover explicit interface implementation, and using the `new` modifier on methods, but it's a good recommendation without going into lots of details in this introductory paragraph.

**Comment by JustNrik** [comment]
> @JustNrik Thanks for opening this, and updating it based on @mairaw 's comments. What you've done has made this better, and has made me notice a couple of issues unrelated to your PR in the same paragraph. Let me know if you want to address these as well in the same PR. If not, I'll open an issue and fix them myself:
> 
> > A class might include an interface multiple times through base classes that it inherits or through interfaces that other interfaces implement.
> 
> The end of this sentence should also be "other interfaces implement."
> 

Hello, I didn't understand well this part, perhaps you meant "other interfaces inherit."?

**Review by BillWagner — APPROVED** [review]
Thanks for making the updating, and especially correctly interpreting my comment. Your :eyes: are better than mine on that.

These changes are great @JustNrik  I'll :shipit: now. You should see the changes on the live site by Monday, on our regular publishing cycle.

Thanks for making your first contribution to docs! We appreciate it.  🎆

**Pull request merged**